docs/changelog.txt

```diff
@@ -1,3 +1,19 @@
+0.5.1 (2020-05-10)
+------------------------------------------------------------------------
+- Add: Builtin functions roadtype() and tramtype()
+- Fix #116: towname used bits could be the same for sub-parts (#117)
+- Fix #120: Value range of transported_last_month_pct should be 0..100 (#122)
+- Fix #121: check StorageOp register range only once
```

Comment:
Maybe something more user-relevant? e.g.

Fix #121: STORE_PERM address limit should be 63